tagging/routes/routes.go

```diff
@@ -85,3 +95,17 @@ func init() {
 		),
 	}.H().Register("/tagging/resources")
 }
+
+func init() {
+	routes.MethodMuxer{
+		http.MethodGet: routes.H(routeGetTaggingSuggestions).With(
+			db.RequestTransaction{db.Db},
+			users.RequireAuthenticatedUser{users.ViewerAsParent},
+			routes.QueryArgs(suggestionsQueryArgs),
+			routes.Documentation{
+				Summary:     "get suggestions for a tag's value",
+				Description: "Responds with suggestions for a tag's value for a user.",
+			},
+		),
+	}.H().Register("/tagging/suggestions")
```

Comment:
Since there will probably be other types of "suggestion" in the future, it would be better to use a more explicit route name. (exemple: `tagging/suggestions/tag-value` That's one example, but I think there's a better way...)